### ISSUE TITLE
If verify fails, `cargo msrv` should have a non-zero return code

### ISSUE THREAD

**Issue by TDHolmes** [foresterre/cargo-msrv#224]
When checking MSRV in CI, it is helpful when a command that fails has a non-zero return code. If it doesn't, we end up having to parse stdout/stderr for strings like Failure. For example, I seem to have to do this in my CI for a project:
```
cargo msrv --verify --output-format json | grep "\"success\":true"
```

I poked around the source code to try to see how to best fix this, but it was hard to follow, so I'm not sure how best to propagate this failure to an eventual `return Err(something);` in `main`. Or `std::process:exit`

**Comment by foresterre** [comment]
Included in [0.13.0](https://crates.io/crates/cargo-msrv/0.13.0).

**Comment by TDHolmes** [comment]
Thanks!!
